packages/web/src/pages/chat-page/components/CreateChatModal.tsx

```diff
@@ -53,24 +66,23 @@ export const CreateChatModal = () => {
     }
   }, [dispatch, isVisible])
 
+  const defaultUserList = useMemo(
```

Comment:
does this... do anything? It's late and I had a thought that maybe this makes a new object each time but I haven't processed in my brain whether that makes a different here or not...